exe/rubocop

```diff
@@ -9,9 +9,7 @@ require 'benchmark'
 cli = RuboCop::CLI.new
 result = 0
 
-time = Benchmark.realtime do
-  result = cli.run
-end
+time = Benchmark.realtime { result = cli.run }
```

Comment:
I think it's better to accept `do ... end` line break by default, because `do ... end` and `{ ... }` have different meanings.

Reply:
Let me check if I understand what you mean. Is it

1. that `do ... end` and `{ ... }` bind differently so changing between them could change the semantics of the code
2. that you might choose one or the other to convey a meaning to the reader, like a value being returned from the block (["Weirich style"](https://github.com/rubocop/ruby-style-guide/issues/162))
3. something else :slightly_smiling_face: 

Becase if it's 1, then it's taken care of by `Style/BlockDelimiters` since `Layout/RedundantLineBreak` only auto-corrects to `time = Benchmark.realtime do result = cli.run end`

Reply:
I agree it's fine to convert the blocks, unless this changes the meaning of the code due to the different precedence.

Reply:
I didn't have enough words 💦  I have two concerns.

1. I'm concerned that it will be an unsafe cop because the operator precedence will change. 
2. It's possible that user intentionally writes it on multi-line instead of single line. IMHO, it feels better to respect readability intended by user by default (I'm not sure about one-liner `{ ... }` may not always be better).

Therefore, it seems safer to configure it to allow multi-line blocks by default.

Reply:
1. In the `convertible_block?` method I put the condition `send_node.parenthesized? || !send_node.arguments?` to make sure the cop only reports blocks where `do ... end` and `{ ... }` behave the same.
2. I agree that a configuration option for blocks would be nice. It's probably as you say that some people may chose `do .. end` because they think it's more readable. @bbatsov Should I add a configuration option now, or can it wait since the cop is disabled by default? @koic Do you see any changes made by `rubocop -a` in this PR where you think that `do ... end` was better?

Reply:
I think it's fine to make changes to the cop later. Cops that are disabled by default rarely get used anyways. Likely we'll be the only user for the cop for a while. :-)